### DISCUSSION TITLE
How to improve text detection models?

### DISCUSSION THREAD

**Discussion by decadance-dance** [mindee/doctr#1495] [Q&A]
Hi everyone. I am interesting how I can improve current text detection models and achieve better results on the FUNSD dataset.
I'd like to fine-tune db_resnet50 and linknet_resnet18 models.
I would be grateful if you could share information what datasets were the detectors trained on? Was it doctr's private data, open source or a combination of them?
If it’s open source, could you also tell what datasets you used?

And of course any other tips or advices that will help me to improve text detection models.
Thanks a lot.

**Comment by felixT2K** [comment]
Hi @decadance-dance 👋🏼,

The detection models are trained on a mindee internal dataset (123073 train samples and 13675 val samples).
What's the background to improve only the FUNSD metrics ? Because this is only a small dataset which gives a little insight into the capacity of the model, but definitely does not describe the overall capacity :)

**Reply by decadance-dance** [reply]
I think FUNSD represents target documents for my tool well. So I suggest if I improve metrics for FUNSD, I will improve quality on my documents too.  
In general, I just want models to work better on my documents. My documents look like samples from FUNSD. I’m not saying that FUNSD is the only or best option that I should focus on, but I haven’t found any similar datasets.
